AirSkyBoat/AirSkyBoat
# Fixed pricing error on Cooking Guild Hare Meat.

**_I affirm:_**


## Please enter a player-facing description

Slightly reduced the price of Hare Meat when purchasing from the Cooking Guild (Hugin)

## What does this pull request do? (Please be technical)

The "min" and "max" prices were swapped causing the price to be stuck at 999,999,999.

## Steps to test these changes

Make sure the guild has some Hare Meat by selling some to them. Wait for the shop to close and re-open so the price gets recalculated. It should be purchasable for between 136 and 139 gil.

## Special Deployment Considerations

Run dbtool to update `modules/era/sql/guild_shops.sql`
